### ISSUE TITLE
CSS Grid cell background geometry doesn't get updated until you hover them

### ISSUE THREAD

**Issue by lobau** [Volumetrics-io/mrjs#313]
This is a CSS grid with 2 columns.

```html
<mr-div class="grid">

  <!-- First column -->
  <mr-div class="section">
      <mr-text>Lorem ipsum…</mr-text>
  </mr-div>
  
  <!-- Second column -->
  <mr-div class="section">
      <mr-div class="illustration">
          <mr-model src="koifish.glb"></mr-model>
      </mr-div>
  </mr-div>

</mr-div>
```

```css
.grid {
    display: grid;
    width: 100%;
    grid-template-columns: 1fr 1fr;
    grid-gap: 3vw;
    justify-items: center;
    align-items: center;
    padding: 6vw;
}
```

Each column is set to fill the whole grid cell

```css
.section {
    display: flex;
    flex-direction: column;
    align-items: start;
    justify-content: center;
    width: 100%;
    height: 100%;
}
```

The right section has a background color. Unfortunately, the background size (and I assume the geometry of the cell) doesn't get updated until you hover the cell.

A video is simpler to explain:
[Screencast from 2024-01-25 17-18-08.webm](https://github.com/Volumetrics-io/mrjs/assets/6750584/25cd24cf-98c4-4aa0-8db6-a78c7103617d)

Here is the sample code:
https://codepen.io/lobau/pen/PoLOqYK

**Comment by hanbollar** [comment]
this should be fixed by #315 , @lobau let me know if it's not

**Comment by hanbollar** [comment]
looks good
